src/notebooks/controllers/controllerLoader.ts

```diff
@@ -75,26 +74,6 @@ export class ControllerLoader implements IControllerLoader, IExtensionSyncActiva
         // Make sure to reload whenever we do something that changes state
         const forceLoad = () => this.loadControllers(true);
         this.serverUriStorage.onDidChangeUri(forceLoad, this, this.disposables);
-        this.serverUriStorage.onDidRemoveUris(
```

Comment:
Moved the logic for this filtering into the controllerRegistration. It has all the other logic for filtering, so figured this would make sense being there.